inductiva/molecules/scenarios/md_water_box/post_processing.py

```diff
@@ -21,20 +20,21 @@ def __init__(self, sim_output_path: Path = None):
 
         self.sim_output_dir = sim_output_path
 
-    def render_interactive(self):
+    def render_interactive(self, use_compressed_trajectory: bool = False):
         """Render the simulation outputs in an interactive visualization."""
 
         topology = os.path.join(self.sim_output_dir, "eql.tpr")
-        trajectory = os.path.join(self.sim_output_dir, "eql.xtc")
-        universe = mda.Universe(topology, trajectory, all_coordinates=True)
-        atoms = universe.atoms
-        transformation = transformations.unwrap(atoms)
-        universe.trajectory.add_transformations(transformation)
+        if use_compressed_trajectory:
+            trajectory = os.path.join(self.sim_output_dir, "trajectory.xtc")
+        else:
+            trajectory = os.path.join(self.sim_output_dir, "eql.trr")
 
+        universe = unwrap_trajectory(topology, trajectory)
         view = nv.show_mdanalysis(universe)
         view.add_ball_and_stick("all")
         view.center()
-        view.parameters = {
-            "backgroundColor": "white"
-        }  # Set the background color
+        view.background = "white"
```

Comment:
Is it possible to wrap this view parameters allocation in a function? It would make this much cleaner

Reply:
I was able to remove the background color and the center, so i think now it is not necessary.